docs/4.0/getting-started/webpack.md

```diff
@@ -26,7 +26,7 @@ import 'bootstrap/js/dist/dropdown';
 ...
 {% endhighlight %}
 
-Bootstrap is dependent on [jQuery](https://jquery.com/) and [Popper](https://popper.js.org/), so npm will install them for you if needed. But they must be explicitly provided by webpack. Add the following code to the `plugins` section in your webpack config file:
+Bootstrap is dependent on [jQuery](https://jquery.com/) and [Popper](https://popper.js.org), so npm will install them for you if needed. But they must be explicitly provided by webpack. Add the following code to the `plugins` section in your webpack config file:
```

Comment:
Plus there's no gain to remove the trailing slash which personally I always want to use.

Reply:
I do not see any good reasons to add an extra character.

Reply:
There are many good reasons. Most of them do not apply here, but still there's no real gain to save that extra character either.

In fact I've been meaning to go through all docs and fix this consistently.